src/midi/fluid_midi.h

```diff
@@ -39,6 +39,7 @@ fluid_midi_event_t *fluid_midi_parser_parse(fluid_midi_parser_t *parser, unsigne
 
 
 #define MAX_NUMBER_OF_TRACKS 128
+#define MAX_NUMBER_OF_CHANNELS 256
```

Comment:
If the player is only used for standard MIDI files, then 16 channels should be enough. If there is the possibility that people could use more channels via some means, then I would vote to make this a dynamic list (maybe with fluid_list)?

Reply:
I agree 16 is probably more than enough, but a user could have lots of instruments player, multiple players, etc. - plus I wanted to be in keeping with MIDI 2.0's 256-track limit. I did want to make it dynamic, though - I'll try a `fluid_list`

Reply:
Oof - I tried using a fluid_list, but it seems beyond my skill, or perhaps it's unworkable to use a fluid_list to hold `int`s. I'll leave this as an int array for now, it's a lot cleaner, at least, and 256 channels is the max we should need.

Reply:
`fluid_midi_event_t` is a structure that resembles MIDI 1.0 events. MIDI 2.0 is a different story, for which we have much more to do. This shouldn't be intermixed.

If one really has multiple players running, the only way of intercepting events I see is by setting a custom callback function for the player. This would intercept events before they're sent out to the synth. Internally, every player would still deal with 0-15 channels.

You cannot use `fluid_list` either, as appending elements would call malloc() from synth context.

Therefore, I'd vote to change max channels to 16.

Reply:
16 it is!